managed-databases/redis/reference-content/ensuring-data-persistence.mdx

```diff
@@ -0,0 +1,55 @@
+---
+meta:
+  title: Ensuring data persistence
+  description: Learn more about how to ensuring data persistence in Managed Database for Redis™.
+content:
+  h1: Ensuring data persistence
+  paragraph: Learn more about how to ensuring data persistence in Managed Database for Redis™.
+tags: databases user redis username password
+categories:
+  - managed-databases
+  - redis
+---
+
+Ensuring data persistence is ensuring that the data from your applications is saved, in memory or on a disk, even when there are no ongoing sessions where the data is treated. The method used to preserve data between sessions, and avoid data loss varies depending on the compute resource you use to treat your data. It also depends on the way your data is stored.
+
+When data is stored in memory, and no data persistence strategy is put in place, the data will be gone any time the compute process is is stopped.
+
+To avoid this, Redis™ inately supports persistence by performing [snapshots](https://redis.io/docs/management/persistence/).
```

Comment:
```suggestion
To avoid this, Redis™ innately supports persistence by performing [snapshots](https://redis.io/docs/management/persistence/).
```

not sure if "innately" is easy to understand, though